cmd/npmExecuteScripts_test.go

```diff
@@ -51,7 +52,7 @@ func (n *npmExecutorMock) FindPackageJSONFilesWithScript(packageJSONFiles []stri
 }
 
 // RunScriptsInAllPackages mock implementation
-func (n *npmExecutorMock) RunScriptsInAllPackages(runScripts []string, runOptions []string, virtualFrameBuffer bool) error {
+func (n *npmExecutorMock) RunScriptsInAllPackages(runScripts []string, runOptions []string, scriptOptions []string, virtualFrameBuffer bool) error {
```

Comment:
Minor: Maybe you can add a check to the mocking implementation, to check if the `scriptOptions` are passed correctly by `npmExecuteScripts`, similar to the checks for the other parameters. And also add a test case covering that below.

Reply:
done